src/python/pants/backend/python/goals/pytest_runner_integration_test.py

```diff
@@ -108,6 +108,9 @@ def _configure_pytest_runner(
     args = [
         "--backend-packages=pants.backend.python",
         f"--source-root-patterns={SOURCE_ROOT}",
+        # NB: Each test file that tests the debug adapter should pick a unique port
+        #  so that different test files can run concurrently without port collisions.
+        "--debug-adapter-port=22335",
```

Comment:
Should we use the fancy pants env car for this?

Reply:
The what now?

Reply:
Oh, the execution slot? Definitely! Good idea